### PR TITLE
TCJA reform files

### PR DESCRIPTION
**Pull request by codykallen**
This PR adds the Senate version of the Tax Cuts and Jobs Act, using the Chairman's mark released 11/9/17. This PR also renames the existing TCJA json to distinguish between the House and Senate versions. 

@MattHJensen @martinholmer @andersonfrailey @Amy-Xu @jdebacker

### CODE REVIEW

**Comment by codykallen** [comment]
Don't merge this yet. Once the exclusion for business income is merged in (Pr #1651), I'll add that provision to this reform file.

**Comment by MattHJensen** [comment]
> Don't merge this yet. Once the exclusion for business income is merged in (Pr #1651), I'll add that provision to this reform file. 

(This would have been a good opportunity for a WIP label!)

File: taxcalc/reforms/TCJA_Senate.json [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        "_II_brk1":
+            {"2018": [[9525, 19050, 9525, 13600, 19050]]},
+        "_II_brk2":
+            {"2018": [[38700, 77400, 38700, 51800, 77400]]},
+        "_II_brk3":
+            {"2018": [[60000, 120000, 60000, 60000, 120000]]},
+        "_II_brk4":
+            {"2018": [[170000, 290000, 145000, 170000, 290000]]},
+        "_II_brk5":
+            {"2018": [[20000, 390000, 195000, 200000, 390000]]},
```
Comment (MattHJensen) [inline taxcalc/reforms/TCJA_Senate.json]: Looks like the singles are missing a zero.

File: taxcalc/reforms/TCJA_Senate.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        "_PT_brk1":
+            {"2018": [[9525, 19050, 9525, 13600, 19050]]},
+        "_PT_brk2":
+            {"2018": [[38700, 77400, 38700, 51800, 77400]]},
+        "_PT_brk3":
+            {"2018": [[60000, 120000, 60000, 60000, 120000]]},
+        "_PT_brk4":
+            {"2018": [[170000, 290000, 145000, 170000, 290000]]},
+        "_PT_brk5":
+            {"2018": [[20000, 390000, 195000, 200000, 390000]]},
```
Comment (MattHJensen) [inline taxcalc/reforms/TCJA_Senate.json]: Also missing a zero for singles.

**Comment by MattHJensen** [comment]
LGTM. Merging. Thanks @codykallen!

**Pull request merged**